app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const serverInfo = require("./db/server.js")
+const {
+  getTopics
+} = require("./db/controllers/data.controller.js")
+
+const app = express();
+
+app.set('view engine', 'ejs')
```

Comment:
Do you know what this is doing here? Is this needed for this ticket? Please remove this line of code 👀